src/style/shared/number-input.less

```diff
@@ -32,10 +32,6 @@
     border-bottom: @hairline solid @item-active;
 }
 
-.number-input:disabled:hover {
```

Comment:
This will probably get covered in my spectrum color branch. Pretty sure I already altered it there